pydust/src/pytypes.zig

```diff
@@ -284,3 +306,31 @@ fn Slots(comptime definition: type, comptime Instance: type) type {
         }
     };
 }
+
+fn BinaryOperator(
+    comptime definition: type,
+    comptime Instance: type,
+    comptime op: []const u8,
+    comptime op_slot: c_int,
+) type {
+    return struct {
+        const slot: c_int = op_slot;
+
+        fn call(pyself: *ffi.PyObject, pyother: *ffi.PyObject) callconv(.C) ?*ffi.PyObject {
+            const func = @field(definition, op);
+            const typeInfo = @typeInfo(@TypeOf(func));
+            const sig = funcs.parseSignature(op, typeInfo.Fn, &.{});
```

Comment:
Is there a way to just parseSelfArg?

Then we could remove the weird switch statement in argCount

Reply:
Probably, but I still think that switch is right - that part of code should handle a single non-struct arg